framework/logging.cpp

```diff
@@ -2147,8 +2147,9 @@ std::string YamlLogger::thread_id_header(int cpu, int verbosity)
 #else
     line = stdprintf("{ logical: %*d, ", thread_core_spacing().logical, info->cpu_number);
 #endif
-    line += stdprintf("package: %d, core: %*d, thread: %d",
-                     info->package_id, thread_core_spacing().core, info->core_id, info->thread_id);
+    line += stdprintf("package: %d, numa_node: %d, module: %*d, core: %*d, thread: %d",
+                      info->package_id, info->numa_id, thread_core_spacing().core, info->module_id,
```

Comment:
In case `numa_id` or `module_id` is not available we can make this fancy and print "N/A" instead. Hopefully people do not treat the output as CSV.